### DISCUSSION TITLE
"Open in new window" from hint mode

### DISCUSSION THREAD

**Discussion by Abb4** [tridactyl/tridactyl#4667] [Q&A] [answered]
First of all, thank you for this amazing extension. Can't live without it ❤

Currently it is possible to open a link in new tab from hint mode using `F`.

It is also possible to `:tabopen` a book mark in a new tab. Are there maybe equivalents of those for opening in a new browser window instead.

If there are, I would never need to touch my mouse again :)

Thanks

**Comment by bovine3dom** [comment, marked as answer]
`;w` for opening starting a hint mode to open hints in new windows. `w` prefills the command line with `:winopen ` which you can use like `:tabopen ` to open windows :)

**Reply by Abb4** [reply]
Exactly what I was looking for. Thanks you!
